Kafka client code examples (#753)
redpanda-data/documentation: docs/development/index.mdx

</head>

Redpanda is Apache Kafka ® API compatible so any client that works with Kafka
will work out of the box with Redpanda.

> In this section we provide an overview of the clients what we have tested, along with some
> example code to demonstrate how to develop producers and consumers, and interact with the
> http proxy and schema registry:

Reviewer: ```suggestion
This section provides an overview of the clients that have been tested, along with some
example code to demonstrate how to develop producers and consumers, and interact with the
HTTP proxy and schema registry:
```
